test/smoke/src/areas/search/search.test.ts

```diff
@@ -29,14 +29,16 @@ export function setup(logger: Logger) {
 
 		it('searches only for *.js files & checks for correct result number', async function () {
 			const app = this.app as Application;
-			await app.workbench.search.searchFor('body');
-			await app.workbench.search.showQueryDetails();
-			await app.workbench.search.setFilesToIncludeText('*.js');
-			await app.workbench.search.submitSearch();
-
-			await app.workbench.search.waitForResultText('4 results in 1 file');
-			await app.workbench.search.setFilesToIncludeText('');
-			await app.workbench.search.hideQueryDetails();
+			try {
+				await app.workbench.search.searchFor('body');
```

Comment:
This still does 2 searches, because searchFor submits the search, right?

Reply:
moved the SearchFor to after setting the includes, that should technically explicitly call search once?

Reply:
I think searchFor is going to search once when putting the text into the input box, then press enter which will search a second time

Reply:
So then should the code for `searchFor` be changed in general to omit the explicit call to submit?

Reply:
That or I guess you could disable search on type in the workspace if that's more reliable. eg by adding a workspace setting to the smoketest repo https://github.com/microsoft/vscode-smoketest-express/blob/main/.vscode/settings.json